watson/common/config/app.py

```diff
@@ -0,0 +1,9 @@
+from . import config as _config
+
+connector_api_base_url = _config("CONNECTOR_API_BASE_URL", default='/api/virtual-assistant/v1')
```

Comment:
These env variables make sense, but could we add to them to the existing common/config package and use that instead? A lot of that stuff we will remove later but the location of the other api's and most of it we will need here I think